packages/jest-core/src/index.ts

```diff
@@ -5,9 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import {getAbqConfiguration} from '@rwx-research/abq';
-
-if (getAbqConfiguration().shouldHideNativeOutput) {
+if (process.env.ABQ_HIDE_NATIVE_OUTPUT) {
```

Comment:
I'd like to circle back around to cleaning up some of these `process.env` calls but seems unimportant for initial release.